packages/origin-247-notary/package.json

```diff
@@ -52,7 +52,8 @@
         "ts-node": "9.1.1",
         "typechain": "5.1.2",
         "typeorm": "0.2.34",
-        "typescript": "4.3.4"
+        "typescript": "4.3.4",
+        "wait-on": "5.3.0"
```

Comment:
is the `wait-on` used somewhere? I cannot find it in changes diff

Reply:
Yep, it is used in package.json, but was missing in deps.